src/measure_theory/function/l1_space.lean

```diff
@@ -598,40 +599,38 @@ end normed_space_over_complete_field
 
 section trim
 
-variables {H α' : Type*} [normed_group H] [measurable_space H]
-  {m m0 : measurable_space α'} {μ' : measure α'}
+variables {H : Type*} [normed_group H] [measurable_space H] [opens_measurable_space H]
+  {m0 : measurable_space α} {μ' : measure α} {f : α → H}
 
-lemma integrable.trim (hm : m ≤ m0) [opens_measurable_space H] {f : α' → H}
-  (hf_int : integrable f μ') (hf : @measurable _ _ m _ f) :
-  @integrable _ _ m _ _ f (μ'.trim hm) :=
+lemma integrable.trim (hm : m ≤ m0) (hf_int : integrable f μ') (hf : @measurable _ _ m _ f) :
+  integrable f (μ'.trim hm) :=
 begin
-  refine ⟨@measurable.ae_measurable α' _ m _ f (μ'.trim hm) hf, _⟩,
+  refine ⟨measurable.ae_measurable hf, _⟩,
   rw [has_finite_integral, lintegral_trim hm _],
   { exact hf_int.2, },
-  { exact @measurable.coe_nnreal_ennreal α' m _ (@measurable.nnnorm _ α' _ _ _ m _ hf), },
+  { exact @measurable.coe_nnreal_ennreal α m _ (@measurable.nnnorm _ α _ _ _ m _ hf), },
 end
 
-lemma integrable_of_integrable_trim (hm : m ≤ m0) [opens_measurable_space H]
-  {f : α' → H} (hf_int : @integrable α' H m _ _ f (μ'.trim hm)) :
+lemma integrable_of_integrable_trim (hm : m ≤ m0) (hf_int : integrable f (μ'.trim hm)) :
   integrable f μ' :=
 begin
   obtain ⟨hf_meas_ae, hf⟩ := hf_int,
   refine ⟨ae_measurable_of_ae_measurable_trim hm hf_meas_ae, _⟩,
   rw has_finite_integral at hf ⊢,
   rwa lintegral_trim_ae hm _ at hf,
-  exact @ae_measurable.coe_nnreal_ennreal α' m _ _
-    (@ae_measurable.nnnorm H α' _ _ _ m _ _ hf_meas_ae),
+  exact @ae_measurable.coe_nnreal_ennreal α m _ _
+    (@ae_measurable.nnnorm H α _ _ _ m _ _ hf_meas_ae),
 end
 
 end trim
 
 section sigma_finite
 
-variables {α' E : Type*} {m m0 : measurable_space α'} [normed_group E] [measurable_space E]
+variables {E : Type*} {m0 : measurable_space α} [normed_group E] [measurable_space E]
   [opens_measurable_space E]
 
-lemma integrable_of_forall_fin_meas_le' {μ : measure α'} (hm : m ≤ m0)
-  [@sigma_finite _ m (μ.trim hm)] (C : ℝ≥0∞) (hC : C < ∞) {f : α' → E} (hf_meas : ae_measurable f μ)
+lemma integrable_of_forall_fin_meas_le' {μ : measure α} (hm : m ≤ m0)
+  [@sigma_finite _ m (μ.trim hm)] (C : ℝ≥0∞) (hC : C < ∞) {f : α → E} (hf_meas : ae_measurable f μ)
```

Comment:
Just to be sure: couldn't also `sigma_finite` benefit from the same treatment as the other ones in this PR?

Reply:
It can also be changed. I forgot this one.